### PR TITLE
Styling

### LINKED ISSUE #46: Styling
Should be using [Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

### LINKED ISSUE #47: Challengeable person list styling
In the list of people to challenge, we want to signify if the spot is challenged by making the selection grayed out and not clickable. Also let the user know if the spot is open. Don't include the user's name who had the spot if that's the case

### PR DESCRIPTION
**Pull request by atareshawty**
🎉 🍻 🎉 🍻 
screen shot 2016-08-21 at 9 30 09 pm

screen shot 2016-08-21 at 9 28 38 pm

screen shot 2016-08-21 at 9 29 16 pm

screen shot 2016-08-21 at 9 30 40 pm
screen shot 2016-08-21 at 9 31 13 pm
screen shot 2016-08-21 at 9 32 22 pm

Fixes #46 
Fixes #47

### CODE REVIEW

**Comment by btmills** [comment]
This is not hideous.
not-bad

**Comment by btmills** [comment]
In the last screenshot (click the button next to the winner) is the entire name clickable (use a `<label>`)?

**Comment by atareshawty** [comment]
Not currently. That does seem better though

File: views/results/show-for-evaluation.handlebars [hunk 1]
```diff
@@ -1,27 +1,41 @@
...
-        class="ResultsEvaluation-submit {{this.spotId}} {{this.resultId}}"
-        type="submit"
-        value="Submit"
+  <h2>**Click the button next to the winner**</h2>
+  <div class="ResultEvaluations">
+    {{#each results}}
+      <div class="ResultsEvaluation {{this.resultId}}">
+        <div class="ResultsEvaluation-header">
+          <p class="ResultsEvaluation-firstNameNumber {{this.resultId}}">
+            <label>
```
Comment (atareshawty) [inline views/results/show-for-evaluation.handlebars]: @btmills This is much better. Good call

**Comment by atareshawty** [comment]
We're also mobile (mostly iPad) friendly!

File: auth/index.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ passport.use(new Strategy((username, password, done) => {
       if (!user) {
         return done(null, false);
       }
-      if (!user.password && !bcrypt.compareSync(password, user.password)) { // eslint-disable-line no-sync
+      if (!bcrypt.compareSync(password, user.password)) { // eslint-disable-line no-sync
```
Comment (atareshawty) [inline auth/index.js]: Nothing wrong here, just want to make this visible because this doesn't really belong in this PR, but needs done. Leaving it though

File: db/queries/results-for-eval.sql [hunk 3]
```diff
@@ -18,5 +18,5 @@ LEFT JOIN
   ) t2
 ON
   t1.id = t2.id
-JOIN performances AS p ON t1.performanceid = p.id
-WHERE p.closeat < now();
+-- JOIN performances AS p ON t1.performanceid = p.id
```
Comment (atareshawty) [inline db/queries/results-for-eval.sql]: This needs to be uncommented...

File: package.json [hunk 4]
```diff
@@ -22,7 +22,6 @@
     "babel-preset-es2015": "^6.13.2",
     "bcrypt": "^0.8.6",
     "body-parser": "~1.13.2",
-    "bootstrap": "^3.3.7",
```
Comment (atareshawty) [inline package.json]: 🎉

File: views/results/show-for-evaluation.handlebars [hunk 5]
```diff
@@ -1,27 +1,41 @@
...
-          <span>{{this.secondName}} </span><input type="radio" name={{this.resultId}} value={{this.secondNameNumber}} />
-        </p>
-      {{/if}}
-      <textarea class="ResultsEvaluation-firstComments {{this.resultId}}"></textarea>
-      <textarea class="ResultsEvaluation-secondComments {{this.resultId}}"></textarea>
-      <input
-        class="ResultsEvaluation-submit {{this.spotId}} {{this.resultId}}"
-        type="submit"
-        value="Submit"
+  <h2>**Click the button next to the winner**</h2>
```
Comment (atareshawty) [inline views/results/show-for-evaluation.handlebars]: Change this message to `Select the winner by clicking`

File: views/static-pages/no-auth.handlebars [hunk 6]
```diff
@@ -1,2 +1,2 @@
 <h1>Hey! You need to be logged in to do that</h1>
-<img class="NoAuth-image" src="/public/images/brutus-login.jpg" />
+<!-- <img class="NoAuth-image" src="/public/images/brutus-login.jpg" /> -->
```
Comment (atareshawty) [inline views/static-pages/no-auth.handlebars]: Just delete this line

**Comment by atareshawty** [comment]
- [ ] Thoroughly checked on staging
